main/ui/src/main/resources/localization_fr.properties

```diff
@@ -0,0 +1,98 @@
+#-------------------------------------------------------------------------------
+# Copyright (c) 2016 Markus Kreusch
```

Comment:
Put your name here, as this is your work. I know this is kind of redundant to the "Contributors" list, but this line indicates who did the most part.